### DISCUSSION TITLE
Navigating console history goes line by line in multiline statements

### DISCUSSION THREAD

**Discussion by etiennebacher** [posit-dev/positron#4905] [Q&A]
I think this is a bug but maybe I missed it in the issue tracker (I looked for "console history") so posting here to be sure.

When I navigate code history in the console, multi-line statements are considered as individual lines when using the "down" arrow key to go towards more recent calls:

[Screencast from 2024-10-04 21-32-36.webm](https://github.com/user-attachments/assets/b7f599ae-d9b7-4ed4-8112-f564acc4b912)

In Rstudio, using the down arrow wouldn't go individually over all lines, which I think is a more desirable behavior.

**Comment by juliasilge** [comment]
I believe this is the same as https://github.com/posit-dev/positron/issues/2430. Want to take a look and see if you have additional info to add?

**Comment by etiennebacher** [comment]
Indeed I had missed it, I don' t have any details to add so closing this. Thank you
